templates/action.epp

```diff
@@ -26,7 +26,11 @@ action(type="<%= $type %>"
 <%}-%>
 <% if $config { -%>
   <% $config.each |$k, $v| { -%>
+  <%- if is_array($v) { -%>
```

Comment:
the `is_array()` method is old, deprecated, Puppet has a built-in matcher:
```suggestion
  <%- if  $v =~ Array { -%>
```

Reply:
Thanks!

Reply:
did you switch back to the old syntax?